account_move_base_import/models/account_move.py

```diff
@@ -273,9 +274,34 @@ def _prepare_insert(self, move, cols):
         Return a copy of statement
         """
         move_copy = move
-        for k, col in move_copy.iteritems():
+        for k, col in move_copy.items():
             if k in cols:
-                move_copy[k] = self._fields[k].convert_to_column(col, None)
+                # In v11, the record is needed to call convert_to_column on a
+                # Monetary field because it must find the currency to apply the
+                # proper rounding. So we mimic the call here as best as we can.
```

Comment:
The same trick than the v11 PR could be applied I suppose.

Reply:
This is outdated, was required only for v11.0 because I kept using direct SQL and v12.0 now uses ORM.